students/morgan/mailprog/bin/mailroom.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import sys
+from mailprog.ui import main_loop
+from mailprog.dataModels import Storage
+from mailprog.functions import read_donors
+
+
+
+if __name__ == "__main__":
+    infile = "mailprog/data/donors.txt"
```

Comment:
so this is the trick -- you are going to need to find a way to find that file when the package is installed in an arbitrary place. I'll talk about that a bit in class.